src/components/AddNicknameForm/AddNicknameForm.js

```diff
@@ -75,47 +74,35 @@ class AddNicknameForm extends React.Component {
   render () {
     const {
       nickname,
-      password,
       submitting,
     } = this.state
 
     return (
       <form
-        className="add-nickname-form"
+        className={styles.addNicknameForm}
         onSubmit={this._handleSubmit}>
         <ValidatedFormInput
-          className="dark"
+          className={styles.thinInput}
+          disabled={this.props.disabled}
           id="AddNickname"
           label="Nickname"
           name="nickname"
           pattern={ircNickPattern}
           patternMessage="Nickname must start with a letter, contain no spaces, and be between 2-30 characters"
           placeholder="Add a nickname..."
+          title={this.props.title}
           type="text"
           value={nickname}
           onChange={this._handleChange}>
-          <InfoBubble header="reminder" id="NickRegisterReminder">
+          <InfoBubble className={bubbleStyles.nicknameInfo} header="reminder" id="NickRegisterReminder">
```

Comment:
`nicknameInfo` should be a part of `AddNicknameForm`'s styles if it's only used by this component.

Reply:
That is much simpler :P